deploy/0-RequestResponseConsumer.ts

```diff
@@ -8,7 +8,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   console.log('0-RequestResponseConsumer.ts')
 
-  if (network.name != 'baobab') {
+  if (network.name == 'localhost') {
```

Comment:
I am not sure, why we had this condition, but I think we should remove it and allow to deploy to any network. 🤔